README.md

```diff
@@ -76,7 +76,18 @@ Here are some resources for learning more about Leptos:
 
 ## `nightly` Note
 
-Most of the examples assume you’re using `nightly` Rust. If you’re on stable, note the following:
+Most of the examples assume you’re using `nightly` Rust.
+To set up your rustup toolchain using nightly and
+add the ability to compile Rust to WebAssembly:
+
+```
+rustup toolchain install nightly --allow-downgrade
```

Comment:
Is `--allow-downgrade` needed here?

Reply:
Yeah I think I agree; @ultrasaurus I'm assuming `--allow-downgrade` snuck in because you copy and pasted it from somewhere (`tailwind` example?) and it's there because I copy and pasted it from somewhere. But I agree with the feedback that it's probably not necessary. Unless there's a good reason for it to be there, could you just remove it and then I'd be happy to merge this!

Reply:
you are correct -- copied from tailwind example.  I'll remove